### PR TITLE
@feature/gitmove

### PR DESCRIPTION
**Pull request by Eler26**

### CODE REVIEW

File: packages/documentation/linux/index.mdx [hunk 1]
```diff
@@ -369,10 +369,35 @@ To merge from another branch
...
+        git mv -k ...
+
+
+* Do nothing; only show what would happen
+
+        git mv -n ...
+
+* Report the names of files as they are moved.
+
+        git mv -v ...
```
Comment (sukiAth) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -v <source> <destination>
```

File: packages/documentation/linux/index.mdx [hunk 2]
```diff
@@ -369,10 +369,35 @@ To merge from another branch
...
+        git mv -f ...
+
+* Skip move or rename actions which would lead to an error condition. An error happens when a source is neither existing nor controlled by Git, or when it would overwrite an existing file unless -f is given.
+
+        git mv -k ...
+
+
+* Do nothing; only show what would happen
+
+        git mv -n ...
```
Comment (sukiAth) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -n <source> <destination>
```

File: packages/documentation/linux/index.mdx [hunk 3]
```diff
@@ -369,10 +369,35 @@ To merge from another branch
...
 
+
+### git move
+command: 
+
+        git mv <source> <destination>
+
+* force renaming or moving of a file even if the target exists
+
+        git mv -f ...
```
Comment (DionysiosKarimpidis) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -f <source> <destination>
```

Comment (Eler26) [inline packages/documentation/linux/index.mdx]: git mv -f <source> <destination>

File: packages/documentation/linux/index.mdx [hunk 4]
```diff
@@ -369,10 +369,35 @@ To merge from another branch
...
+
+* Skip move or rename actions which would lead to an error condition. An error happens when a source is neither existing nor controlled by Git, or when it would overwrite an existing file unless -f is given.
+
+        git mv -k ...
```
Comment (DionysiosKarimpidis) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -k <source> <destination>
```

File: packages/documentation/linux/index.mdx (hunk 3 above)
Comment (Eler26) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -f <source> <destination>
```

File: packages/documentation/linux/index.mdx (hunk 4 above)
Comment (Eler26) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -k <source> <destination>
```

File: packages/documentation/linux/index.mdx (hunk 2 above)
Comment (Eler26) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -n <source> <destination>
```

File: packages/documentation/linux/index.mdx (hunk 1 above)
Comment (Eler26) [inline packages/documentation/linux/index.mdx]:
```suggestion
        git mv -v  <source> <destination>
```

**Pull request merged**